### DISCUSSION TITLE
ScriptReference/GUI.TextArea

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#257] [Notes]
# ScriptReference/GUI.TextArea

https://docs.unity3d.com/ScriptReference/GUI.TextArea.html

**Comment by radiatoryang** [comment]
If you want cut/copy/paste support in editor, make sure you use `EditorGUI.TextArea` or `EditorGUILayout.TextArea` (warning: on these Editor classes, you can't access TextEditor without reflection)
